hammerblade/torch/tests/test_cat.py

```diff
@@ -0,0 +1,24 @@
+"""
+tests of simple cat kernel.
+"""
+
+import torch
+
+def _test_torch_cat(x):
+    x_h = x.hammerblade()
+    y = torch.cat([x, x, x], 0)
+    y_h = torch.cat([x_h, x_h, x_h], 0)
+    assert y_h.device == torch.device("hammerblade")
+    assert torch.allclose(y, y_h.cpu())
+
+def test_cat_1():
+    x = torch.ones(10)
+    _test_torch_cat(x)
+
+def test_cat_2():
+    x = torch.randn(3, 4)
+    _test_torch_cat(x)
+
+def test_cat_3():
+    x = torch.randn(3, 4, 5)
+    _test_torch_cat(x)
```

Comment:
More tests will make this even better ;)

For instance, I think you would want to test the case where you cat tensors with different length and empty tensors. Say, [0,1], [0,1,2,3,4,5], [] -> [0,1,0,1,2,3,4,5]